src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+interface Props {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<Props> = ({ comments }) => {
+  return comments.length > 0
```

Comment:
you can make it simple
```suggestion
  return comments.length
```